components/account/home/AccountInformation.vue

```diff
@@ -18,44 +19,78 @@ const state = reactive({
   vatId: '',
 });
 
-const loading = ref(false);
-const done = ref(false);
-const vatSuccessMessage = ref('');
+const loading = ref<boolean>(false);
+const done = ref<boolean>(false);
+const vatSuccessMessage = ref<string>('');
+const vatErrorMessage = ref<string>('');
+const loadingCheck = ref<boolean>(false);
+const $externalResults = ref<Record<string, string[]>>({});
+const remainingTime = ref<number>(0);
 
 const { account } = storeToRefs(store);
 
-const movedOffline = computed(
-  () => get(account)?.address.movedOffline ?? false,
-);
-
-const isVatIdValid = computed(
-  () => get(account)?.vatIdStatus === VatIdStatus.VALID || false,
-);
-
-onBeforeMount(() => {
-  reset();
-});
-
 const rules = {
   firstName: { required },
   lastName: { required },
   companyName: {},
   vatId: {},
 };
 
-const $externalResults = ref<Record<string, string[]>>({});
+const waitUntilTime = useLocalStorage<number>('rotki.vat_check.wait_until_time', 0);
```

Comment:
Shouldn't we make it account specific?

Reply:
It would make sense just to be safe.

On the other hand I wonder how many users would have multiple site accounts where they would need to validate the VAT ID?

Reply:
I believe it's a very rare case tho :P

Reply:
I am gonna keep it as it is for now